extensions/cornerstone-3d/src/services/ViewportService/Viewport.ts

```diff
@@ -112,12 +182,24 @@ class ViewportInfo {
   }
 
   private validateDisplaySetOptions(
```

Comment:
Where is the "validate" part of this, it just seems to map the options, no validation is done.